languages/csharp/exercises/concept/inheritance/.docs/introduction.md

```diff
@@ -0,0 +1,59 @@
+In C#, a _class_ hierarchy can be defined using _inheritance_, which allows a derived class (`Car`) to inherit the behavior and data of its parent (general) class (`Vehicle`). If no parent is specified, the class inherits from the `object` class.
+
+The `protected` access modifier restricts access to code in the same (parent) class or in any derived class. The `base` keyword can be used to reference parent class members.
```

Comment:
```suggestion
C# provides control over what parts of a parent class can be accessed by a derived class. The `protected` access modifier allows derived classes to access the parent's code, but blocks access from other classes. The `base` keyword can be used to reference parent class members.
```

> The `base` keyword can be used to reference parent class members.

I suggest moving this below the bullet points and say "Derived classes can access parent class members through the `base` keyword". I think it will fit more contextually there.

As a general thing, I think explaining what's happening before the keyword is mentioned helps with comprehension (e.g. "Derived classes can access parent class members through the `base` keyword" is clearer to me than "The `base` keyword allows derived classes to access parent class members"). My brain feels calmer getting an english explanation before a technical word.

Reply:
Great suggestions. I've rephrased.